static/katex.less

```diff
@@ -442,10 +443,10 @@
     }
 
     .delimsizing {
-        &.size1 { font-family: KaTeX_Size1; }
```

Comment:
Do you mean why I gave `size1` a line-height? For consistency, and because I don't even know half what I'm doing. :grin: